cluster/src/main/java/org/apache/iotdb/cluster/config/ClusterConfig.java

```diff
@@ -127,6 +125,34 @@
    */
   private int readDataConsistencyLevel = 1;
 
+  /**
+   * Max num of threads can concurrently execute qp sub-tasks of all client requests. When <= 0, use
+   * CPU core number * 10. Each client request corresponds to a QP Task. A QP task may be divided
+   * into several sub-tasks.
+   */
+  private int concurrentQPSubTaskThread = Runtime.getRuntime().availableProcessors() * 10;
+
+  /**
+   * Max num of threads can concurrently apply tasks in data partition state machine. When <= 0, use
+   * CPU core number.Raft Task refers to tasks that need to be performed in the state machine.
+   */
+  private int concurrentRaftTaskThread = Runtime.getRuntime().availableProcessors();
+
+  /**
+   * Max time of blocking main thread for waiting for all running task threads and tasks in the
```

Comment:
Is this param necessary? We always have to wait for a log to be executed.

Reply:
What does `a weak writing consistency level` mean? 
If a task takes too much time to execute, you mark it as failed. Does this have anything to do with weak consistency?

Reply:
If the user sets this parameter as a low number, it's will not guarantee consistency. But if the use can tolerate inconsistency，he can set the parameter. On the contrary, if the user wants our system to guarantee consistency, he should not set the parameter. It depends on user's demand.

Reply:
It may not be necessary for users to set such detailed system operating parameters.